### PR TITLE
Feat: automate

### PR DESCRIPTION
**Pull request by oxedions**

### CODE REVIEW

**Review by neilmunday — COMMENTED** [review]
Only minor, but it would be better to say: "The tool expects"

File: roles/automate/README.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+### Logs
+
+Global logs can be found in `/var/log/bluebanquise/automate/global.log`.
+When a task is performed with a specific target node as argument (assuming here 'mynode'), logs can be found in 
+`/var/log/bluebanquise/automate/mynode.log`.
+
+### Triggering tasks
+
+The tool is listening via Flask on port **5000** on **localhost**.
```
Comment (neilmunday) [inline roles/automate/README.md:69]: The tool listens via Flask...

**Review by neilmunday — COMMENTED** [review]
Can the port be adjusted? I only ask as port 5000 is usually used for UPnP.

**Comment by oxedions** [comment]
> Can the port be adjusted? I only ask as port 5000 is usually used for UPnP.

That is a good question, I didn't thought about that. I will open an issue.

**Comment by oxedions** [comment]
I created a small PR to fix typo issues you pointed out 🙂 
https://github.com/bluebanquise/community/pull/50

**Pull request merged**